codalab/lib/bundle_cli.py

```diff
@@ -3311,6 +3311,7 @@ def do_wrm_command(self, args):
         help=[
             'Append all the items of the source worksheet to the destination worksheet.',
             'Bundles that do not yet exist on the destination service will be copied over.',
+            'Bundles in non-terminal states (READY or FAILED) will not be copied over to destination worksheet.'
```

Comment:
This will add a newline in the generated doc
```suggestion
            'Bundles in non-terminal states (READY or FAILED) will not be copied over to destination worksheet.',
```